recipes-core/busybox/busybox/defconfig

```diff
@@ -78,10 +81,12 @@ CONFIG_NO_DEBUG_LIB=y
 # CONFIG_EFENCE is not set
 
 #
-# Busybox Library Tuning
+# Library Tuning
 #
 # CONFIG_FEATURE_USE_BSS_TAIL is not set
+CONFIG_FLOAT_DURATION=y
 CONFIG_FEATURE_RTMINMAX=y
+CONFIG_FEATURE_RTMINMAX_USE_LIBC_DEFINITIONS=y
```

Comment:
```suggestion
# CONFIG_FEATURE_RTMINMAX_USE_LIBC_DEFINITIONS is not set
```
based on this [file](https://github.com/openembedded/openembedded-core/blob/dunfell/meta/recipes-core/busybox/busybox/defconfig#L88).

Reply:
It was enabled by the `make oldconfig` operation in busybox because we had the parent `CONFIG_FEATURE_RTMINMAX` enabled already. Codesearching around for which signal definitions we use suggests that both NI and OE-upstream prefer to use the LIBC definitions. So we should keep this option enabled (which is the default anyway).

@gratian ACKed that assessment.